frame/ddc-staking/Cargo.toml

```diff
@@ -13,15 +13,22 @@ sp-runtime = { version = "6.0.0", default-features = false, path = "../../primit
 sp-staking = { version = "4.0.0-dev", default-features = false, path = "../../primitives/staking" }
 sp-std = { version = "4.0.0-dev", default-features = false, path = "../../primitives/std" }
 
+frame-benchmarking = { version = "4.0.0-dev", default-features = false, path = "../benchmarking", optional = true }
+
+[dev-dependencies]
+substrate-test-utils = { version = "4.0.0-dev", path = "../../test-utils" }
+
 [features]
 default = ["std"]
 std = [
   "codec/std",
   "frame-support/std",
   "frame-system/std",
+  "frame-benchmarking/std",
```

Comment:
Should be `"frame-benchmarking?/std"` (`?` is missing) for an optional dependency.

Reply:
I can't find examples of frame-benchmarking as optional dependency in other frames